### DISCUSSION TITLE
Welcome to Open-source-practice-and-resources Discussions!

### DISCUSSION THREAD

**Discussion by Ashish-khanagwal** [Ashish-khanagwal/Open-Source-Practice-and-Resources#4] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Ashish-khanagwal** [comment]
Hey Contributors👋, Thank you for contributing here. Open Source is for everyone. keep hustling and Keep contributing🔥
Don't forget to share your Contributions on your socials so that more folks can start their Open Source journey with us.🙌

**Reply by tillytillie** [reply]
Hello Ashish thank for this opportunity, I just joined the community and I hope I will make a tangible and contribution to the community in this coming days. 

Thanks once again for this wonderful ideal.. ❤️

**Reply by Ashish-khanagwal** [reply]
Welcome to the communtiy Tilly. It's just a small contribution from my side to the community. It's so good seeing folks like you coming forward to grab the opportunities.
Thank you for joining us :smile:

**Reply by tillytillie** [reply]
I need resources on API integration in JS because I always encounter difficulties in implementing API.

**Reply by Ashish-khanagwal** [reply]
we'll add it soon. Well, if you have any resources you can try adding those resources as well. If you find any difficulty join our discord server and ask your queries on how to contribute.

**Comment by harshhes** [comment]
Hello, i just made a PR to updated README changes, "Add new resources #103" . I want to know is my PR still open as PR says that 1 review is still required

**Reply by Ashish-khanagwal** [reply]
I'll review your PR then it'll get merged. until then its remain open
